lib/internal/crypto/util.js

```diff
@@ -270,6 +250,44 @@ const kSupportedAlgorithms = {
   },
 };
 
+const experimentalAlgorithms = {
+  'X25519': {
+    generateKey: null,
+    importKey: null,
+    deriveBits: 'EcdhKeyDeriveParams',
+  },
+  'Ed25519': {
+    generateKey: null,
+    sign: null,
+    verify: null,
+    importKey: null,
+  },
+  'X448': {
+    generateKey: null,
+    importKey: null,
+    deriveBits: 'EcdhKeyDeriveParams',
+  },
+  'Ed448': {
+    generateKey: null,
+    sign: 'Ed448Params',
+    verify: 'Ed448Params',
+    importKey: null,
+  },
+};
+
+for (const { 0: name, 1: ops } of ObjectEntries(experimentalAlgorithms)) {
+  for (const { 0: op, 1: dict } of ObjectEntries(ops)) {
```

Comment:
```suggestion
const experimentalAlgorithms = ObjectEntries({
  'X25519': {
    generateKey: null,
    importKey: null,
    deriveBits: 'EcdhKeyDeriveParams',
  },
  'Ed25519': {
    generateKey: null,
    sign: null,
    verify: null,
    importKey: null,
  },
  'X448': {
    generateKey: null,
    importKey: null,
    deriveBits: 'EcdhKeyDeriveParams',
  },
  'Ed448': {
    generateKey: null,
    sign: 'Ed448Params',
    verify: 'Ed448Params',
    importKey: null,
  },
});

for (let i = 0; i < experimentalAlgorithms.length; i++) {
  const name = experimentalAlgorithms[i][0];
  const ops = ObjectEntries(experimentalAlgorithms[i][1]);
  for (let j = 0; j < ops.lentgh; j++) {
    const { 0: op, 1: dict } = ops[j];
```

https://github.com/nodejs/node/blob/daea065f249648a005a9caaa548a76db42eacbb1/doc/contributing/primordials.md#L151-L213

Reply:
Thank you, i'll take a look